### ISSUE TITLE
When first modal opens, the ghost keeps boo-ving 👻 

### ISSUE THREAD

**Issue by anibalsolon** [proxygo/O-fantasma-Detetive#2]
Fun lil game! One thing I noticed:

When the "Buuuuuuuuuuuuuuh!" modal opens, the ghost keeps its speed, which may bother some since they lost control of the game.

Ideally, the game would pause (freeze); when the modal is closed, the ghost will return to their speed i.e. no change on dx & dy) This behaviour could be achieved by pausing 'Atualizar' while the modal is showing.

**Comment by proxygo** [comment]
great idea, but I don't know how to implement this, I'm a beginner and I don't know how to fix this.  Would you have a suggestion for a solution? =)

**Comment by anibalsolon** [comment]
Yeah, it turns out it could be done with an even simpler solution- I've opened a [PR](https://github.com/proxygo/O-fantasma-Detetive/pull/3) for this :smiley_cat:
